.dockerignore

```diff
@@ -0,0 +1,3 @@
+**/.gitignore
```

Comment:
So we need these at root and /web?

Reply:
I'm honestly not sure. They're being handled differently on linux and on mac - which is causing me some confusion. I'm really not sure when they get used and how - I'll play around a bit. Hopefully we can get rid of one.